test/gtest/uct/ib/test_cqe_zipping.cc

```diff
@@ -68,7 +68,8 @@ class test_cqe_zipping : public test_uct_ib_with_specific_port {
     virtual void init()
     {
         stats_activate();
-        modify_config("IB_CQE_ZIPPING_ENABLE", "y");
+        modify_config("IB_TX_CQE_ZIP_ENABLE", "yes");
```

Comment:
@ivankochin 
Can we feasibly test the RX / TX zipping separately and independently?

Reply:
I think we can, but let me please do it as a separate patch.